shared/constant/constant.go

```diff
@@ -33,7 +33,7 @@ var DefaultConfigOrder = [...]string{
 
 const (
 	// DefaultCloneImage can be changed by 'WOODPECKER_DEFAULT_CLONE_IMAGE' at runtime
-	DefaultCloneImage = "docker.io/woodpeckerci/plugin-git:2.1.1"
+	DefaultCloneImage = "quay.io/woodpeckerci/plugin-git:2.1.1"
```

Comment:
Why should we change this one? I would prefer to stick with the old one.

Reply:
I've just picked up the outdated PR and fixed conflicts. If you want to stay with docker.io by default fine for me.

Reply:
Would like to stay with it for now. To me using `docker.io` or `quay.io` makes no difference both are some "random" companies hosting images for free.